src/main.tsx

```diff
@@ -13,14 +13,15 @@ import DeviceData from './views/DeviceData';
 import ConnectEhr from './views/ConnectEhr';
 import ExternalAccounts from "./views/ExternalAccounts";
 import TaskProgress from './components/TaskProgress/TaskProgress';
-
+import MyDataHelps from '@careevolution/mydatahelps-js';
+ 
 if (window.location.hostname === "localhost") {
-	window.MyDataHelps.setParticipantAccessToken({ "access_token": process.env.REACT_APP_PARTICIPANT_ACCESS_TOKEN, "expires_in": 21600, "token_type": "Bearer" }, "https://mydatahelps.org/");
+	MyDataHelps.setParticipantAccessToken({ "access_token": process.env.REACT_APP_PARTICIPANT_ACCESS_TOKEN ?? "", "expires_in": 21600, "token_type": "Bearer" }, "https://mydatahelps.org/");
```

Comment:
This line is still failing for me attempting to use a valid participant access token in development mode. I'm getting "Uncaught ReferenceError: process is not defined".

It looks like there's some extra setup to get env fully working with Vite: https://vitejs.dev/guide/env-and-mode.html#env-files